scripts/env

```diff
@@ -0,0 +1,304 @@
+#!/usr/bin/env bash
```

Comment:
as we now have shasum for this script published, should we also download it instead?